dwh/es_data_marts/models/dataset_summary.sql

```diff
@@ -6,13 +6,21 @@
         on_schema_change="sync_all_columns",
```

Comment:
Number of datasets (1210) matches 1194 CRISPR + 10 MAVE + 6 Perturb-Seq. However, as noted in a comment for `mave_data.sql`, it's truncated. This means that `dataset_summary` also only contains a portion of the MAVE datasets, not all of them.